auth_tkt/ticket.py

```diff
@@ -20,15 +24,15 @@ def validate(ticket, secret, ip='0.0.0.0', timeout=7200):
 
     if '!' not in ticket:
         try:
-            raw = ticket.decode('base64')
+            raw = base64decode(ticket, encoding)
             base64 = True
         except binascii.Error:
             return False
 
     if '!' not in raw:
         return False
 
-    digest, raw = raw[:32], raw[32:]
+    raw = raw[32:]
```

Comment:
Correct, we validate the digest by checking the return value of `AuthTkt.ticket` below, if it matches what is passed here, the digest is valid.